python/ml4ir/base/features/feature_fns/categorical.py

```diff
@@ -369,3 +300,117 @@ def categorical_indicator_with_vocabulary_file(feature_tensor, feature_info, fil
     categorical_one_hot = tf.expand_dims(categorical_one_hot, axis=1)
 
     return categorical_one_hot
+
+
+def get_categorical_indices_from_vocabulary(feature_info, feature_tensor, file_io):
```

Comment:
@Ullimague Can we rename this to `get_categorical_indices_from_vocabulary_file`?